website/docs/advanced-concepts/version-control-with-git/environments-with-git.md

```diff
@@ -1,41 +1,60 @@
-# Environments With Git
+# Multi-instance setup with Git 
 
-Appsmith enables you to create and maintain multiple versions of your applications and make changes to them in isolation using Git. Having separate staging and production workspaces ensures that users can continue to use your apps even while they're being changed or upgraded.
+This guide shows you how to set up multiple instances/environments in Appsmith using Git, making application development easier. Appsmith allows you manage multiple app versions, making development streamlined with separate staging and production workspaces.
 
 :::info
 This method doesn't support APIs, Google Sheets, Twilio, or any authenticated API datasources.
```

Comment:
@harshilp24  - can you get this verified with Engineering once, I think Authenticated APIs work now. @AnaghHegde ?